terasoluna-gfw-common-libraries/pom.xml

```diff
@@ -1,150 +1,160 @@
+    <module>terasoluna-gfw-codepoints/catalog/terasoluna-gfw-codepoints-jisx0208kanji</module>
+    <module>terasoluna-gfw-codepoints/catalog/terasoluna-gfw-codepoints-jisx0213kanji</module>
+  </modules>
+  <build>
+    <pluginManagement>
+      <plugins>
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-surefire-plugin</artifactId>
+          <version>${maven-surefire-plugin.version}</version>
+          <configuration>
+            <includes>
+              <include>**/*Test.java</include>
+            </includes>
+            <argLine>${jacocoArgs}</argLine>
+          </configuration>
+        </plugin>
+        <plugin>
+          <groupId>org.jacoco</groupId>
+          <artifactId>jacoco-maven-plugin</artifactId>
+          <version>${jacoco-maven-plugin.version}</version>
+          <executions>
+            <execution>
+              <id>prepare-agent</id>
+              <phase>test-compile</phase>
+              <goals>
+                <goal>prepare-agent</goal>
+              </goals>
+              <configuration>
+                <propertyName>jacocoArgs</propertyName>
+                <includes>
+                  <include>*</include>
+                </includes>
+              </configuration>
+            </execution>
+          </executions>
+        </plugin>
+        <plugin>
+          <groupId>com.google.code.maven-license-plugin</groupId>
+          <artifactId>maven-license-plugin</artifactId>
+          <version>${com.google.code.maven-license-plugin.version}</version>
+          <configuration>
+            <header>${project.root.basedir}/../license/header.txt</header>
+            <includes>
+              <include>src/**/*.java</include>
+            </includes>
+            <encoding>${encoding}</encoding>
+            <headerDefinitions>
+              <headerDefinition>${project.root.basedir}/../license/header-definition.xml</headerDefinition>
+            </headerDefinitions>
+          </configuration>
+        </plugin>
+        <plugin>
+          <groupId>net.revelc.code.formatter</groupId>
+          <artifactId>formatter-maven-plugin</artifactId>
+          <version>${formatter-maven-plugin.version}</version>
+          <configuration>
+            <configFile>${project.root.basedir}/../eclipse/formatter.xml
+            </configFile>
+            <lineEnding>CRLF</lineEnding>
+            <directories>
+              <directory>${project.basedir}</directory>
+            </directories>
+            <includes>
+              <include>**/src/main/java/**/*.java</include>
+              <include>**/src/test/java/**/*.java</include>
+            </includes>
+            <excludes>
+              <exclude>terasoluna-gfw-string/src/main/java/org/terasoluna/gfw/common/fullhalf/DefaultFullHalf.java</exclude>
+            </excludes>
+          </configuration>
+        </plugin>

```

Comment:
I added `xml-maven-plugin` setting at `terasoluna-gfw-common-libraries/pom.xml` and `terasoluna-gfw-dependencies/pom.xml` at XXX commit .